product_brand/views/product_brand_view.xml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="product_brand_search_form_view" model="ir.ui.view">
+        <field name="name">product.brand.search.form</field>
+        <field name="model">product.brand</field>
+        <field name="arch" type="xml">
+            <search string="Product Brand">
+                <field name="name" />
+                <field name="partner_id" />
+            </search>
+        </field>
+    </record>
+    <act_window
+        id="action_open_brand_products"
+        name="Brand Products"
+        res_model="product.template"
+        view_mode="kanban,form,tree"
+        domain="[('product_brand_id', '=', active_id)]"
+    />
+    <act_window
+        id="action_open_single_product_brand"
+        name="Product Brand"
+        res_model="product.brand"
+        view_mode="kanban,form,tree"
+        target="current"
+        domain="[('product_ids', 'in', active_id)]"
+    />
```

Comment:
The `act_window` has been deprecated. You should migrate to the `record` tag. You can use [this](https://github.com/OCA/product-attribute/pull/742/commits/40eca7185f6a8cfb02d3dde10a841a84f92c9178#diff-68f52c9b5cbb5cad326ecb91bf58f75aa8b5a47d4b82f88b0e1dd3b575ea6df7L96-R141) as a reference.